nb-engine/nb-engine-core/src/main/java/io/nosqlbench/engine/core/lifecycle/process/NBCLIErrorHandler.java

```diff
@@ -43,7 +44,7 @@ public class NBCLIErrorHandler {
 
     private final static Logger logger = LogManager.getLogger("ERRORHANDLER");
 
-    public static String handle(Throwable t, boolean wantsStackTraces, String version) {
+    public static int handle(Throwable t, boolean wantsStackTraces, String version) {
```

Comment:
I'm not sure what the benefit of this change is except it makes this class more "posix"

Reply:
It's due to the new early exit handling which is either an error or not.